### ISSUE TITLE
Dead code in metric_wrapper.cc (Coverity CID 80092)

### ISSUE THREAD

**Issue by jacobkeeler** [smartdevicelink/sdl_core#739]
[Coverity Report](https://scan9.coverity.com/reports.htm#v26942/p12036/fileInstanceId=7040670&defectInstanceId=1401215&mergedDefectId=80092)

Coverity found a piece of dead code in [MetricWrapper::grabResources()](https://github.com/smartdevicelink/sdl_core/blob/master/src/components/telemetry_monitor/src/metric_wrapper.cc#L48), where for some reason the code attempts to flush stdout after returning a value.

``` cpp
if (NULL != resources) {
  return true;
} else {
  return false;
}
flush(std::cout);
```

I don't know exactly what this command is supposed to accomplish, but it isn't doing so successfully. Is there any actual reason why this line was included? 

If there isn't, this should be a quick fix of just removing the `flush(std::cout)` line.

**Comment by jacobkeeler** [comment]
Could be fixed along with #407

**Comment by jacobkeeler** [comment]
Fixed in master
